src/main/scala/togyzkumalak/format/Forsyth.scala

```diff
@@ -38,9 +38,12 @@ object Forsyth {
 
   def <<(fen: FEN): Option[Situation] = <<@(Variant.default, fen)
 
-  case class SituationPlus(situation: Situation, fullMoveNumber: Int) {
+  case class SituationPlus(situation: Situation, fullTurnCount: Int) {
+
+    def turnCount = fullTurnCount * 2 - situation.player.fold(2, 1)
+    // when we get a multiaction variant we should set this
```

Comment:
suppose we can keep the comment in case we copy this module for another game and that required multiaction